components/merkledb/src/lib.rs

```diff
@@ -194,3 +194,7 @@ pub mod proto;
 
 #[cfg(feature = "with-protobuf")]
 use exonum_proto::ProtobufConvert;
+
+#[cfg(test)]
+#[macro_use]
+extern crate assert_matches;
```

Comment:
Why is this declaration required? In the 2018 edition, macros could be exported from a crate just like other items.